sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -362,13 +362,23 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder(conf) {
       } else {
         Map.empty[String, String]
       }
-      DescribeTableCommand(
-        visitTableIdentifier(ctx.tableIdentifier),
+      DescribeTableOrQueryCommand(
+        Option(visitTableIdentifier(ctx.tableIdentifier)),
```

Comment:
`visitTableIdentifier` can return null?

Reply:
@maropu with the change, DescribeTableOrQueryCommand takes a optional table identifier. Thats why i wrap it under an `Option`.

Reply:
oh, I see.